### PR TITLE
Replace deprecated (P)Config functions with new syntax

### PR DESCRIPTION
**Pull request by zeroadam**
Replace deprecated functions with new syntax, cf https://github.com/friendica/friendica/issues/3873

### CODE REVIEW

File: twitter/twitter.php [hunk 1]
```diff
@@ -642,12 +645,12 @@ function twitter_cron($a,$b) {
 	$r = q("SELECT * FROM `pconfig` WHERE `cat` = 'twitter' AND `k` = 'mirror_posts' AND `v` = '1'");
 	if(count($r)) {
 		foreach($r as $rr) {
-			logger('twitter: fetching for user '.$rr['uid']);
+			logger('twitter:PConfig::get fetching for user '.$rr['uid']);
```
Comment (annando) [inline twitter/twitter.php]: Could you remove the falsely added text here?

**Comment by zeroadam** [comment]
Removed

**Comment by zeroadam** [comment]
Replaced load_pconfig with PConfig::load()

**Comment by tobiasd** [comment]
del_pconfig was forgotten to be replaced, see #442

**Pull request merged**